### PR TITLE
Ensure pa11y demo exists to run tests against

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by rowanmanning** [comment]
Tests are failing due to some trouble with saucelabs by the looks

**Comment by JakeChampion** [comment]
@rowanmanning Fixed.

File: circle.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 machine:
   node:
-    version: 8.6.0
+    version: 8
```
Comment (alexflorisca) [inline circle.yml]: Why 8 and not 8.6?

Comment (JakeChampion) [reply circle.yml]: I believe Circle will use the latest version of node 8 that is released.

Comment (alexflorisca) [reply circle.yml]: 👍

**Comment by alexflorisca** [comment]
I think `obt verify` is failing at the moment on circle

**Comment by alexflorisca** [comment]
I just realised how big this PR is, I thought you'd only changed the circle.yml file haha! Having a proper look now

**Pull request closed without merging**